website/content/api-docs/system/storage/raft.mdx

```diff
@@ -39,19 +39,6 @@ leader node.
 - `leader_client_key` `(string: "")` - Client key used to communicate with
   Raft's leader node.
 
-- `auto_join` `(string: "")` - Defines any cloud auto-join metadata. If
```

Comment:
Did you mean to revert this part?

Reply:
Yep. I wanted to keep this a clean revert. I'll PR that separately.